v0.12.0/siteinfo.js

```diff
@@ -0,0 +1 @@
+var DOCUMENTER_CURRENT_VERSION = "dev";
```

Comment:
```suggestion
var DOCUMENTER_CURRENT_VERSION = "v0.12.0";
```
I think this should be the released version instead of `dev`.

Reply:
Yes, indeed. This was somehow not staged.